src/Kmd.Logic.CitizenDocuments.Client/CitizenDocumentsClient.cs

```diff
@@ -89,6 +94,111 @@ public async Task<CitizenDocumentUploadResponse> UploadAttachmentWithHttpMessage
             }
         }
 
+        /// <summary>
+        /// Uploads the single citizen document.
+        /// </summary>
+        /// <param name="configurationId">Citizen document provider config id.</param>
+        /// <param name="retentionPeriodInDays">Retention period of the uploaded document.</param>
+        /// <param name="cpr">Citizen CPR no.</param>
+        /// <param name="documentType">Type of the citizen document.</param>
+        /// <param name="document">Original citizen document.</param>
+        /// <param name="documentName">Preferred name of citizen document.</param>
+        /// <param name="citizenDocumentUploadRequestModel">citizenDocumentUploadRequestModel to update to db.</param>
+        /// <returns>The fileaccess page details or error if isn't valid.</returns>
+        /// <exception cref="ValidationException">Missing cpr number.</exception>
+        /// <exception cref="SerializationException">Unable to process the service response.</exception>
+        /// <exception cref="LogicTokenProviderException">Unable to issue an authorization token.</exception>
+        /// <exception cref="CitizenDocumentsException">Invalid Citizen document configuration details.</exception>
+        public async Task<CitizenDocumentUploadResponse> UploadLargeFileAttachmentWithHttpMessagesAsync(Guid configurationId, int retentionPeriodInDays, string cpr, string documentType, IFormFile document, string documentName, CitizenDocumentUploadRequestModel citizenDocumentUploadRequestModel)
```

Comment:
ConfigurationId , retentionPeriodInDays, Cpr, documentType, documentName already in citizenDocumentUploadRequestModel . What is the need for using as separate parameter.

Reply:
Removed the parameters which are part of request model